### PR TITLE
databases/week1

### PR DESCRIPTION
**Pull request by Jul-S**

### CODE REVIEW

File: databases/week1/homework-queries.sql [hunk 1]
```diff
@@ -0,0 +1,80 @@
+-- Find out how many tasks are in the task table
+SELECT
+  COUNT(id)
```
Comment (calindurnea) [inline databases/week1/homework-queries.sql]: You should create a new line per statement, e.g.
`SELECT COUNT(*) as NumberOfTasks`
`FROM task;`

File: databases/week1/homework-queries.sql [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+SELECT
+  status.name,
+  COUNT(task.id)
+FROM
+  task
+  JOIN status on task.status_id = status.id
+GROUP BY
+  task.status_id
+ORDER BY
+  COUNT(task.id) DESC;
```
Comment (calindurnea) [inline databases/week1/homework-queries.sql]: In the SELECT statement you can set `COUNT(task.id) as TaskCount` and then you can use it in the ordery by statement `ORDER BY TaskCount DESC`

**Review by calindurnea — COMMENTED** [review]
Well done 💯 ! just a couple of comments 👍

File: databases/week1/homework-queries.sql (hunk 2 above)
Comment (Jul-S) [reply databases/week1/homework-queries.sql]: Thank you! i`ve updated PR

**Review by calindurnea — APPROVED** [review]

**Review by calindurnea — APPROVED** [review]

**Pull request merged**